Makefile.PL

```diff
@@ -0,0 +1,11 @@
+use ExtUtils::MakeMaker;
+
+os_unsupported unless $^O eq 'linux';
```

Comment:
It looks like the [AssertOS](https://metacpan.org/pod/Dist::Zilla::Plugin::AssertOS) might be the Dist::Zilla way of doing this. Maybe like the following?

```
[AssertOS]
os = Linux
```

This would also mean we don't need to keep a manually generated Makefile.PL and we can offload that to Dist::Zilla (which seems to be the point of using it in the first place).

Reply:
Added.